src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,78 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int ACTION = 0;
+    private static final int NUMBER = 1;
+    private static final int SUPPLIED = 0;
+    private static final int BOUGHT = 1;
+    private static final int RESULT = 2;
+    private static final int NUMBER_OF_ROWS = 3;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT_STR = "result";
+    private static final String COMA = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readFile(fromFileName);
+        writeInFile(resultTextMaker(statisticCalculator(readFile(fromFileName))), toFileName);
```

Comment:
You call `readFile()` method but not saving the result of its work, and in line 24 you call `readFile()` method again